msu/systems/mod_settings/settings_element.nut

```diff
@@ -16,7 +16,7 @@
 		}
 		this.Name = _name == null ? _id : _name;
 		this.ID = _id;
-		this.Flags = [];
+		this.Flags = this.new("scripts/tools/tag_collection");;
```

Comment:
I don't see any reason to use flag collection instead of a table for the changes you're making here.